### DISCUSSION TITLE
Dynamic styles??

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [chakra-ui/panda#750] [Q&A] [answered]
Hi there, i'm kinda curious how this works in terms of dynamic styles e.g. 

```
const loggedIn = props.auth.isAuthorized

<div className={css({ fontSize: 'lg', color: loggedIn ? 'red.400' : 'blue.400' })}>Box 2</div>
```

How would this work in terms of build time, as this isnt something i would think could be done at build time but maybe im wrong?

**Comment by anubra266** [comment, marked as answer]
@The-Code-Monkey https://panda-css.com/docs/guides/dynamic-styling#runtime-conditions

**Reply by segunadebayo** [reply]
tldr; Simple conditionals like this will work in Panda.

When it gets complex, you want to consider using a cva recipe instead.
